index.html

```diff
@@ -129,10 +129,12 @@ <h1 id="title">Decentralized Identifier Working Group Charter</h1>
 
         <p>
 URL identifiers in use on the Web today (2019) require that the identifier is
```

Comment:
This change shouldn't be in here. Looks like you branched from your other commit?

Reply:
addressed in [1fd80c1]9https://github.com/w3c-ccg/did-wg-charter/pull/28/commits/1fd80c1636204dba15ddcead66183ad4187b870c)